### DISCUSSION TITLE
Add a tiny line in the README to help begginers out

### DISCUSSION THREAD

**Discussion by spiderman-idog** [usememos/memos#893] [Ideas]
I'm super noob at Docker, so when I read the instructions to deploy in Docker run:

> ## Docker Run
> `docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest`
> If the `~/.memos/` does not have a `memos_prod.db` file, then memos will auto generate it. Memos will be running at http://localhost:5230/.

I thought I could set the `/var/opt/memos` folder to something else (I thought this was going to be a folder being used on my file system, not the container file system), so I simply set it was a folder inside my `home` directory, something like this:

`-v ~/.memos/:/home/username/memos_opt`

That was a big mistake, the database files didn't appear in `~/.memos`, nor in `/var/opt/memos`. (Not sure if that's a bug though :stuck_out_tongue:)

Later I learned that the database files were created inside the container file system, but that took quite a bit of digging to find out.

So my humble suggestion is to add a tiny line in that README snippet above of a little heads up telling to only edit the first argument, like:

> ## Docker Run
> `docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest`
> You can edit the `~/.memos/` directory to something else, but do not modify the `/var/opt/memos` directory.
> If the `~/.memos/` does not have a `memos_prod.db` file, then memos will auto generate it. Memos will be running at http://localhost:5230/.

Sounds like a silly request, but I think it would help inexperienced people out.

**Comment by boojack** [comment]
@Helaxious SGTM, could you create a PR for it

**Reply by spiderman-idog** [reply]
Nice, making a PR right away.

**Reply by spiderman-idog** [reply]
Ok, done
